idaes/power_generation/unit_models/boiler_heat_exchanger.py

```diff
@@ -110,9 +127,14 @@ def delta_temperature_underwood_tune_callback(b):
     This uses a cube root function that works with negative numbers returning
     the real negative root. This should always evaluate successfully.
     """
-    temp_units = pyunits.get_units(b.deltaT_1)
-    dT1 = b.deltaT_1
-    dT2 = b.deltaT_2
+    _log.warning(
+        "DEPRECATED: delta_temperature_underwood_tune_callback will be "
+        "removed. Use another standard delta_temperature callback or "
+        "provide a custom callback.  This will be removed in IDAES 3.0."
```

Comment:
IDAES 3.0 seems a bit far in the future (seeing as we haven't got a concrete date for IDAES 2.0 yet). Would IDAES 2.1 be OK? (still vague, but sooner).

Reply:
I'm not sure when 2 will be, but I figure it has to be a major release, and I want to leave a lot of time before I break anything.  I'm open to other numbers though this will mostly affect @MAZamarripa and Jinliang.  I'll happily put whatever in there.